srml/council/src/lib.rs

```diff
@@ -14,7 +14,159 @@
+//! - Proposing, vetoing, and passing motions.
+//!
+//! The Council's role is to represent passive stakeholders. The Council is an on-chain entity comprised of
+//! a set of account IDs. Its primary tasks are to propose sensible referenda and thwart any uncontroversially
+//! dangerous or malicious referenda.
+//!
+//! ### Terminology
+//!
+//! #### Council Proposals
+//!
+//! - **Council motion:** A mechanism used to enact a proposal.
+//! - **Council origin:** The council (not root) that contains the council motion mechanism.
+//! - **Proposal validity:** A council proposal is valid when it's unique, hasn't yet been vetoed, and
+//! when the council term doesn't expire before the block number when the proposal's voting period ends.
+//! - **Proposal postponement:** Councillors that abstain from voting may postpone a council proposal from
+//! being approved or rejected. Postponement is equivalent to a veto, which only lasts for the cooloff period.
+//! - **Cooloff period:** Period, in blocks, for which a veto is in effect.
+//!
+//! #### Council Proposal Voting
+//!
+//! - **Proposal:** A submission by a councillor. An initial vote of yay from that councillor is applied.
+//! Unlike the Democracy and Treasury modules, the `Proposal` type is very generic.
+//! - **Referendum:** The means of voting on a proposal.
+//! - **Vote:** A vote of yay or nay from a councillor on a single proposal. Councillors may change their vote.
+//! - **Veto:** A council member may veto any council proposal that exists. A vetoed proposal that's valid is set
+//! aside for a cooloff period. The vetoer cannot re-veto or propose the proposal again until the veto expires.
+//! - **Vote cancellation:** At the end of a given block we cancel all referenda that have been
+//! elevated to the Table of Referenda whose voting period ends at that block and where the outcome of the vote
+//! tally was a unanimous vote to cancel the referendum.
+//! - **Voting process to elevate a proposal:** At the end of a given block we tally votes for expiring referenda.
+//! Referenda that are passed (yay votes are greater than nay votes plus abstainers) are sent to the Democracy
+//! module for a public referendum. If there are no nay votes (abstention is acceptable), then the proposal is
+//! for immediate enactment. Otherwise, there will be a delay period. If the vote is unanimous, then the public
+//! referendum will require a vote threshold of supermajority against to prevent it. Otherwise,
+//! it is a simple majority vote.
+//!
+//! #### Council Seats
+//!
+//! - **Desired seats:** The number of seats on the council. Can change via governance.
+//! - **Candidacy bond:** Bond required to be a candidate.
+//! - **Voting bond:** Bond required to be permitted to vote. Must be held because many voting operations affect
+//! storage. The bond is held to disincent abuse.
+//! - **Candidate approval voting call:** Express candidate approval voting is a public call that anyone may execute
```

Comment:
I see this is a bit strange, both the title and the text. I can understand one thing from it: there is a signed call that triggers approval voting procedure, which is not the case (and probably I misread it).

Reply:
Yeah, there is a signed call that basically says "I have votes for myself that should be included", then there is another call (automatically at some block number) that produces the outcome.